### PR TITLE
支持回调函数处理异常内容

### PR DESCRIPTION
**Pull request by 0523**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Please both format this PR correctly as per the docs, and send this to the correct repo.

**Comment by GrahamCampbell** [comment]
I doubt this will be accepted anyways, because this looks like a bad idea.

File: ExceptionHandler.php [hunk 1]
```diff
@@ -201,6 +201,13 @@ public function getHtml($exception)
      */
     public function getContent(FlattenException $exception)
     {
+        /**
+         * 支持回调函数处理异常内容
```
Comment (GrahamCampbell) [inline ExceptionHandler.php]: Symfony requires all comments to be written in English.

**Comment by GrahamCampbell** [comment]
> I doubt this will be accepted anyways, because this looks like a bad idea.

**Comment by GrahamCampbell** [comment]
> I doubt this will be accepted anyways, because this looks like a bad idea.

Well, the changing the visibility of the properties is at least.

It may be worth submitting the public method changes to the correct place.

**Comment by Lisennk** [comment]
Why not to use English?

**Comment by GrahamCampbell** [comment]
Sorry, but this repo is read only, and this change is probably a bad idea?

**Comment by lyrixx** [comment]
Please, re-open your pull request on https://github.com/symfony/symfony

But I doubt that this PR will be accepted.

**Pull request closed without merging**